crates/services/sync/src/import.rs

```diff
@@ -58,17 +53,20 @@ mod back_pressure_tests;
 #[derive(Clone, Copy, Debug)]
 /// Parameters for the import task.
 pub struct Config {
-    /// The maximum number of get header requests to make in a single batch.
-    pub max_get_header_requests: usize,
     /// The maximum number of get transaction requests to make in a single batch.
     pub max_get_txns_requests: usize,
+    /// The maximum number of headers to request in a single batch.
+    pub header_batch_size: u32,
```

Comment:
Why is this `u32` while the other amounts are `usize`? To me it seems like `get_headers_buffered` is doing unnecessary type conversions because of this.